phpBB/phpbb/avatar/manager.php

```diff
@@ -268,9 +268,16 @@ public function get_avatar_settings($driver)
 	{
 		$config_name = $driver->get_config_name();
 
-		return array(
-			'allow_avatar_' . $config_name	=> array('lang' => 'ALLOW_' . strtoupper(str_replace('\\', '_', $config_name)),		'validate' => 'bool',	'type' => 'radio:yes_no', 'explain' => true),
-		);
+		return [
+			'allow_avatar_' . $config_name	=> [
```

Comment:
Thanks for restoring the radio:yes_no and enabled_disabled options. Can you please revert this code to use the radio:yes_no again?